src/library.c

```diff
@@ -93,6 +93,7 @@ static struct event *updateev;
 // Counts the number of changes made to the database between to DATABASE
 // event notifications
 static unsigned int deferred_update_notifications = 0;
+static short deferred_update_events = 0;
```

Comment:
You don't need to set to zero, globals are always initialized to that. So I would recommend removing it from the above one as well.